src/components/policy/policy_regular/include/policy/policy_table/types.h

```diff
@@ -67,7 +67,7 @@ typedef Array<Enum<HmiLevel>, 0, 4> HmiLevels;
 
 typedef Array<Enum<Parameter>, 0, 24> Parameters;
 
-typedef Map<RpcParameters, 0, 50> Rpc;
+typedef Map<RpcParameters, 0, 65535> Rpc;
```

Comment:
@LuxoftAKutsan yes, not just big. We don't have any limitations regarding rpc count, so it should be unlimited. Replaced with macro. See https://github.com/smartdevicelink/sdl_core/pull/1610/commits/d4aa621768d568d6dc563fbdf620155875bd6257